mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php

```diff
@@ -66,11 +69,13 @@ public function test_should_return_all_stored_configs_when_no_key_or_configs() {
 			],
 		];
 		foreach ( $configs as $store_key => $config ) {
-			_the_store( $store_key, $config );
+			loadConfig( $store_key, $config );
```

Comment:
@rgadon107 You don't want to use functions from the API if you can avoid it. Why? What happens if `loadConfig` breaks? Also `_the_store` is used inside of `loadConfig`. 

Instead, keep `_the_store` here.

Reply:
@hellofromtonya Interesting. I hadn't considered the possibility of `loadConfig()` breaking. Thought using it would make the code more readable. Will revert changes.